ws_builds/robot_ws.sh

```diff
@@ -0,0 +1,4 @@
+rosws update
+rosdep install --from-paths src --ignore-src -r -y
+colcon build --build-base build --install-base install
+colcon bundle --build-base build --install-base install --bundle-base bundle
```

Comment:
Missing newline at the end as per [POSIX 2.306](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206). (from a practical standpoint  this would cause some weird inconsistencies w.r.t how different GNU utilities process the file)